hash/msx2_flop.xml

```diff
@@ -12215,6 +12288,33 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="sailorfk">
+		<description>Sailor Fuku Senshi Ferisu (Japan)</description>
+		<year>1990</year>
+		<publisher>Cocktail Soft</publisher>
+		<info name="alt_title" value="セーラー服戦士フェリス"/>
```

Comment:
Transliterated title should be “Sailor-fuku Senshi Felis”.

Reply:
Thank you, I was unsure how to do the 'Ferisu' part so I had kept it as-is.